oauthdoctor/diag/config.go

```diff
@@ -188,8 +188,10 @@ func (c *ConfigFile) ReplaceConfigFromReader(key, value string, r io.Reader) str
 		langKey := c.GetConfigKeysInLang(key)
 
 		// Found the line with old config key and comment it out
-		if c.Lang == "dotnet" && strings.Contains(trimmedLine, langKey) {
-			buf.WriteString("<!-- " + trimmedLine + " -->\n")
+		if c.Lang == "dotnet" {
+			if strings.Contains(trimmedLine, langKey) {
+				buf.WriteString("<!-- " + trimmedLine + " -->\n")
```

Comment:
Why are you using the comment literal here if you already know that the lang is dotnet?